docs/BitcoinCoreKeys.md

```diff
@@ -0,0 +1,120 @@
+# Using Eclair to manage your Bitcoin Core wallet's private keys
+
+You can configure Eclair to control (and never expose) the private keys of your Bitcoin Core wallet. This feature was designed to take advantage of deployements where your Eclair node runs in a
+"trusted" runtime environment, but is also very useful if your Bitcoin and Eclair nodes run on different machines for example, with a setup for the Bitcoin host that
+is less secure than for Eclair (because it is shared among several services for example).
+
+## Configuring Eclair and Bitcoin Core to use a new Eclair-backed bitcoin wallet
+
+Follow these steps to delegate onchain key management to eclair:
+
+1) Generate a BIP39 mnemonic code and passphrase
+
+You can use any BIP39-compatible tool, including most hardware wallets.
+
+2) Create an `eclair-signer.conf` configuration file add it to eclair's data directory
```

Comment:
Using a new `signer` section is `eclair.conf` is simpler but "leaks" because it becomes available everywhere in the code through the actor system's settings instead of being contained to where it is actually needed and could potentially be exposed through a bug that "exports" the configuration (through logs, ....) through this is highly unlikely (especially if eclair runs in a environment where we explicitly control all I/O).

I don't feel very strongly about this and using a new section in `eclair.conf` would be a very simple change though.

Reply:
Those are good reasons to keep it separate :+1: , it's maybe worth a comment in the code to explain the rationale